source/_integrations/cloudflare.markdown

```diff
@@ -7,7 +7,7 @@ ha_category:
 ha_release: 0.74
 ---
 
-With the `cloudflare` integration you can keep your Cloudflare records up to date.
+With the `cloudflare` integration you can keep your Cloudflare records up to date. This integration uses python package [pycfdns](https://github.com/ludeeus/pycfdns) developed by [Joakim Sørensen](https://github.com/ludeeus).
```

Comment:
We don't mention the dependencies a integration has. This is tracked in the `manifest.json` file. Please remove it.

Reply:
Sure